nixos/modules/services/security/fprot.nix

```diff
@@ -16,16 +16,19 @@ in {
           description = ''
             product.data file. Defaults to the one supplied with installation package.
           '';
+          type = types.path;
```

Comment:
```suggestion
          type = types.either types.path types.str;
```

Reply:
I was asked by @cole-h whether this makes sense to allow both paths and derivations. However this is in fact not necessary, `types.path` works for both paths and derivations already, so I think this change can be reverted, same for `licenseKeyfile` :)

Reply:
Sorry, @Scriptkiddi :grimacing: